### PR TITLE
Added "Deletion Scheduled" to bandwidth pools and pool-details

### PR DESCRIPTION
**Pull request by edsonarios**
Issue: #2002 
```
softlayer-python/ (issue2002) $ slcli bandwidth pools
                                                            Bandwidth Pools
┌─────────┬──────────────────────┬─────────────────┬─────────┬────────────┬───────────────┬─────────────────┬──────┬──────────────────┐
│ Id      │ Name                 │ Region          │ Devices │ Allocation │ Current Usage │ Projected Usage │ Cost │ Deletion         │
├─────────┼──────────────────────┼─────────────────┼─────────┼────────────┼───────────────┼─────────────────┼──────┼──────────────────┤
│ 12928   │ Test_8_Region        │ IND             │ 0       │ 0 GB       │ 0 GB          │ 0 GB            │ $0.0 │ 2023-07-03 22:59 │
└─────────┴──────────────────────┴─────────────────┴─────────┴────────────┴───────────────┴─────────────────┴──────┴──────────────────┘
```
```
softlayer-python/ (issue2002) $ slcli bandwidth pools-detail 12928
┌──────────────────┬──────────────────┐
│             name │ value            │
├──────────────────┼──────────────────┤
│               Id │ 12928            │
│             Name │ Test_8_Region    │
│      Create Date │ 2023-06-02 00:00 │
│         End Date │ 2023-07-03 22:59 │
│    Current Usage │ None GB          │
│ Projected  Usage │ 0 GB             │
│  Inbound   Usage │ 0 GB             │
│         hardware │ Not Found        │
│    virtualGuests │ Not Found        │
│        Netscaler │ Not Found        │
└──────────────────┴──────────────────┘
```

### CODE REVIEW

**Review by caberos — APPROVED** [review]
LGTM

**Comment by ramkishor-ch** [comment]
HI,

I have tested the changes,
Please find the screen shots for reference.

Screen Shot:
Screenshot 2023-06-30 at 5 51 03 PM

Thanks,
Ramkishor Chaladi.

**Review by ramkishor-ch — APPROVED** [review]

File: SoftLayer/CLI/bandwidth/pools.py [hunk 1]
```diff
@@ -55,5 +56,7 @@ def cli(env):
         else:
             cost = "$0.0"
 
-        table.add_row([id_bandwidth, name, region, servers, allocation, current, projected, cost])
+        deletion = utils.clean_time(item.get('endDate'))
```
Comment (allmightyspiff) [inline SoftLayer/CLI/bandwidth/pools.py:59]: Add a blank value if this is empty

**Review by allmightyspiff — CHANGES_REQUESTED** [review]

**Comment by allmightyspiff** [comment]
This pull request will be handled in https://github.com/softlayer/softlayer-python/pull/2006

**Pull request closed without merging**